crema/parsers/msfragger.py

```diff
@@ -0,0 +1,104 @@
+"""A parser for the MSFragger tab-delimited format"""
+import re
+import logging
+
+import pandas as pd
+
+from .txt import read_txt
+from .pepxml import read_pepxml
+from .pepxml import _parse_pepxml
+from ..dataset import PsmDataset
+from .. import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def read_msfragger(txt_files, pairing_file_name=None, copy_data=True):
+    """Read peptide-spectrum matches (PSMs) from MSFragger tab-delimited files.
+
+    Parameters
+    ----------
+    txt_files : str, pandas.DataFrame or tuple of str
+        One or more collection of PSMs in the MSFragger tab-delimited format.
+    pairing_file_name : str, optional
+        A tab-delimited file that explicity pairs target and decoy peptide
+        sequences. Requires one column labled 'target' that contains target
+        sequences and a second colun labeled 'decoy' that contains decoy
+        sequences.
+    copy_data : bool, optional
+        If true, a deep copy of the data is created. This uses more memory, but
+        is safer because it prevents accidental modification of the underlying
+        data. This argument only has an effect when `txt_files` is a
+        :py:class:`pandas.DataFrame`
+
+    Returns
+    -------
+    PsmDataset
+        A :py:class:`~crema.dataset.PsmDataset` object containing the parsed
+        PSMs.
+    """
+    target = "label"
+    peptide = "peptide"
+    # TODO well annoying the column names for pepXML and tsv output are different
+    # spectrum = ["Filename", "start scan"] # check this - is this for TSV?
+    spectrum = ["ms_data_file", "scan"]
+    pairing = ""
+    # protein = "protein" # check for TSV
+    protein = "proteins"
+    protein_delim = ";"
+    decoy_prefix = "rev_"
```

Comment:
**question:** Is this always "rev_"?

Reply:
That is the default value. There is a parameter though. Will add an argument.